packages/cli/test/mocks/deployment.ts

```diff
@@ -26,13 +46,15 @@ export function useDeployment({
     version: 2,
     createdAt,
     createdIn: 'sfo1',
+    buildingAt: Date.now(),
     ownerId: creator.id,
     creator: {
       uid: creator.id,
       email: creator.email,
       username: creator.name,
     },
     readyState: 'READY',
+    state: states[Math.floor(Math.random() * states.length)],
```

Comment:
I prefer to not use randomness in tests. It can often be fine, but will sometime cause a test to pass or fail unpredictably.

Reply:
Agreed. Also, why would state and readyState be different?